### PR TITLE
ExtractDestination is disabled to have vsolutions size > 1

### PR DESCRIPTION
**Pull request by dimxy**
disable vSolutions size > 1 after June2021 HF as it might break addressindex (this was mistakenly pulled from verus code)

### CODE REVIEW

File: src/komodo_utils.h [hunk 1]
```diff
@@ -2497,3 +2497,8 @@ void komodo_prefetch(FILE *fp)
     }
     fseek(fp,fpos,SEEK_SET);
 }
+
+bool komodo_Is2021JuneHFActive()
+{
+    return GetLatestTimestamp(komodo_currentheight()) > nS5Timestamp;
```
Comment (ca333) [inline src/komodo_utils.h:2503]:
```suggestion
    return GetLatestTimestamp(komodo_currentheight()) > nS5HardforkHeight;
```

Comment (ca333) [inline src/komodo_utils.h:2503]: any specific reason why we use timestamp as opposed to block height check? As you know for Komodo blockchain it is `komodo hardfork-height != hardfork-timestamp` and difference can be significant

Comment (DeckerSU) [reply src/komodo_utils.h:2503]: I guess timestamp check only is used bcz this change is related to CC, and CC are active only in smartchains for now. But may be will be better to define `komodo_Is2021JuneHFActive` as:
```
bool komodo_Is2021JuneHFActive(uint32_t time)
{
    return ( (ASSETCHAINS_SYMBOL[0] == 0 && chainActive.Height() > nS5HardforkHeight) || (ASSETCHAINS_SYMBOL[0] != 0 && time > nS5Timestamp) );
}
```

Comment (dimxy) [reply src/komodo_utils.h:2503]: this fix is supposed for cc chains (maybe it is better to rename the komodo_Is2021JuneHFActive() function to clarify it is timestamp based)

Comment (dimxy) [reply src/komodo_utils.h:2503]: this function was designed as self-contained, with no params, as it is called only one-time to fix one specific potential issue in standard.cpp (where we do not have neither chain time nor height)

Comment (DeckerSU) [reply src/komodo_utils.h:2503]: i see. just offered a universal function, that can be used in other places too if needed. bcz currently we haven't any other function to determin is the S5 hardfork active or not.

Comment (Alrighttt) [reply src/komodo_utils.h:2503]: the code is not active on any chains other than ac_cc, so it's safe to do only the timestamp activation IMO

**Comment by dimxy** [comment]
I am having build problems on linux with this PR - do not merge!!
(it looks like a linker undef reference error - libbitcoin_common_a lib where standard.cpp resides cannot refer to symbols in the libbitcoin_server lib)

**Comment by dimxy** [comment]
updated the PR, added a time-activation function stub in the utils to allow the utils to build

**Review by ca333 — APPROVED** [review]

**Pull request merged**